### ISSUE TITLE
Add ability to sign in and persist stats across devices

### ISSUE THREAD

**Issue by jimmyfang94** [rbtying/shengji#425]

**Comment by rbtying** [comment]
Yeah, this one's interesting. I kind of like that the activation energy to develop the game is low (no true persistent storage). But putting in actual logins would require putting actual state in.

Or, perhaps, there's a simple free way to do this that isn't too much work to deploy...
